pkg/convert/xmb.go

```diff
@@ -0,0 +1,71 @@
+package convert
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+type xmb struct {
+	XMLName  xml.Name     `xml:"messagebundle"`
+	Language language.Tag `xml:"srcLang,attr"`
+	Messages []msg        `xml:"message"`
+}
+
+type msg struct {
+	ID          string `xml:"id,attr"`
+	Description string `xml:"desc,attr"`
+	Content     string `xml:"content"`
+	Fuzzy       bool   `xml:"fuzzy,attr"`
+}
+
+func FromXMB(b []byte) (model.Messages, error) {
```

Comment:
```suggestion
func FromXmb(b []byte) (model.Messages, error) {
```
Consistency with other From's